x-pack/plugins/apm/common/agent_configuration/setting_definitions/java_settings.ts

```diff
@@ -175,7 +176,7 @@ export const javaSettings: RawSettingDefinition[] = [
       'xpack.apm.agentConfig.profilingInferredSpansEnabled.description',
       {
         defaultMessage:
-          'Set to `true` to make the agent create spans for method executions based on async-profiler, a sampling aka statistical profiler. Due to the nature of how sampling profilers work, the duration of the inferred spans are not exact, but only estimations. The `profiling_inferred_spans_sampling_interval` lets you fine tune the trade-off between accuracy and overhead. The inferred spans are created after a profiling session has ended. This means there is a delay between the regular and the inferred spans being visible in the UI. This feature is not available on Windows'
+          'Set to `true` to make the agent create spans for method executions based on async-profiler, a sampling (aka statistical profiler). Due to the nature of how sampling profilers work, the duration of the inferred spans are not exact, but only estimations. The `profiling_inferred_spans_sampling_interval` lets you fine tune the trade-off between accuracy and overhead. The inferred spans are created after a profiling session has ended. This means there is a delay between the regular and the inferred spans being visible in the UI.\n\nNOTE: This feature is not available on Windows.'
```

Comment:
I get what you're going for, but I don't the the parenthesis work in this case. Also, do the newlines actually render?

Here's a parenthesis suggestion:

```suggestion
          'Set to `true` to make the agent create spans for method executions based on async-profiler, a sampling (or statistical) profiler. Due to the nature of how sampling profilers work, the duration of the inferred spans are not exact, but only estimations. The `profiling_inferred_spans_sampling_interval` lets you fine tune the trade-off between accuracy and overhead. The inferred spans are created after a profiling session has ended. This means there is a delay between the regular and the inferred spans being visible in the UI.\n\nNOTE: This feature is not available on Windows.'
```

Reply:
Let's restore the former - `a sampling aka statistical profiler.`. I thought it would be better with parentheses, but seems you think it won't, so I will restore.
The main change I wanted here is the separation of the `NOTE`. I don't know about these newlines, I just saw what you did in other descriptions and did the same 🙂 . Since you found they do not render, I will remove... 
So currently we have no formatting for these sections whatsoever?

Reply:
Ok, sounds good! Correct, there's unfortunately no formatting